README.md

```diff
@@ -7,8 +7,6 @@ The script looks for the next subtitle and if it is ahead by 5 (default) seconds
 ### Usage
 For the script to work it is necessary to have an appropriate 'text' subtitle selected and visible.
 
-If you want to use it with local files it is necessary to either have `--cache=yes` or `--demuxer-readahead-secs=10` options enabled in your config/cli. A value of `demuxer-readahead-secs>=10` is recommended. The same applies to `--cache-secs` option if it has been set.
```

Comment:
This is now automatically set with 
mp.set_property("demuxer-readahead-secs",lookahead+leadin)